model/api/src/main/java/org/jboss/aerogear/unifiedpush/utils/AeroGearLogger.java

```diff
@@ -46,23 +46,23 @@ public void info(String message){
     }
 
     public void warning(String message){
-        AeroGearLogger.logger.log(Level.WARNING, format(message));
+        AeroGearLogger.logger.warn(format(message));
```

Comment:
just wondering, w/ slf4j do we still need the custom `format()` ? I hope/think that's done by the slf4j logging library